apps/ashley/templates/services/app/dc.yml.j2

```diff
@@ -32,9 +32,10 @@ spec:
                       values:
                         - "ashley-app-{{ deployment_stamp }}"
                 topologyKey: kubernetes.io/hostname
-{% if ashley_image_pull_secret_name is defined and ashley_image_pull_secret_name is not none %}
+{% set image_pull_secret_name = ashley_image_pull_secret_name or default_image_pull_secret_name %}
+{% if image_pull_secret_name is defined and image_pull_secret_name is not none %}
```

Comment:
If it is set, it will always be defined right?

Reply:
```suggestion
{% if image_pull_secret_name is not none %}
```

Reply:
true... :pray: